framework/src/org/checkerframework/framework/util/AnnotationBuilder.java

```diff
@@ -97,54 +97,65 @@ private void assertNotBuilt() {
     public AnnotationMirror build() {
         assertNotBuilt();
         wasBuilt = true;
-        return new AnnotationMirror() {
```

Comment:
`org.checkerframework.javacutil.AnnotationUtils#fromName` also uses an anonymous inner class.

Reply:
Yes, but that use is from a static method. So the anonymous class there can't capture an enclosing instance and can't cause an object to stick around.
But I'll look into moving everything into javacutils and having only one class that creates AnnotationMirrors.

Reply:
Oh right!